themes/scss/widgets/default.scss

```diff
@@ -26,12 +39,17 @@
   margin-right: $sMargin-section;
   margin-left: $sMargin-section;
 }
+.Widget-header--timeSeries {
```

Comment:
I wasn't aware of that component, I'll have a look to see how I would go about changing this then.